src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -282,6 +282,19 @@ function isDisabled(node) {
     return node.disabled ? true : false;
 }
 
+function hasAutofocus(node) {
```

Comment:
Just to drop in: @githubert What is the state of the recheck-web-js? Shouldn't this be done there`?

Reply:
As long as this hasn't been finished, we need to keep them in sync. But from my understanding, this is a minimal change in the JS.

Reply:
Then I would suggest that you create an corresponding PR in recheck-web-js? Just that this is not lost when transitioning. 

Just to clarify: I do not know what the best approach is here. In my understanding, this was already in progress. But let's wait what @githubert thinks.

Reply:
An additional PR would be nice, so I can implement it over there. Once I figure out how to make the Internet Explorer happy, it'll be ready. (Last status was that it successfully runs the script and extracts the attributes, but then it goes wrong _somewhere_ in the whole process)

Reply:
Or no PR, but open an issue.

Reply:
Done: https://github.com/retest/recheck-web-js/issues/61